ros/processor/insights_engine_result_consumer.py

```diff
@@ -116,6 +130,26 @@ def process_report(self, host, reports):
                     number_of_recommendations=rec_count,
                     state=SYSTEM_STATES[state_key]
                 )
+                LOG.info(
+                    f"{self.prefix} - System created/updated successfully: {host['id']}"
```

Comment:
For consistency, let's either use `self.prefix` or `custom_prefix` globally.

Reply:
The custom_prefix's default assignment was implemented such when `get_performance_profile` gets called
from the engine consumer, the logs messages start with `PROCESSING ENGINE RESULTS`
instead the archive processor's prefix i.e. `ARCHIVE PROCESSOR`

Changes [here](https://github.com/RedHatInsights/ros-backend/pull/138/files#diff-6255fd2253fb40ee086cf8feeb2c42da4450a36667cd5e6405801fe4e81e2e0fR59-R60)

In the commented file's case the self.prefix seems to do its job, or am I missing something here :sweat_smile:

Reply:
So IIUC and based on a quick look at the code, we do want the log to have `ARCHIVE PROCESSOR` prefix here as that is where these log statements reside.
```python
2021-10-14 19:40:51,195 - INFO  - get_performance_profile - ARCHIVE PROCESSOR - Downloading the report from http://minio:9000/insights-upload-perma/<url continues with credentials>.
```

Reply:
As discussed on Slack, keeping this change in place.